chapters/packages.tex

```diff
@@ -195,6 +195,12 @@ \subsection{Example of Searching MODELICAPATH}\label{example-of-searching-modeli
 hierarchy to the file system.  The subpackage \lstinline!Pack2! is stored in its own subdirectory or file in the subdirectory \filename{MyLib}.  In this case the search
 succeeds and the package \lstinline!MyLib.Pack2! is loaded into the environment.
 \end{example}
+
+
+If during lookup a top-level name is not found in the unnamed top-level
+scope, the search continues in the package hierarchies stored in these
+directories.
```

Comment:
This paragraph is now duplicated. It was part of a section that moved in master-branch.

Reply:
Well spotted!  Fixed.  Thanks!